app/views/components/_global_bar.html.erb

```diff
@@ -11,7 +11,7 @@
   coronavirus_href = "/coronavirus"
   coronavirus_subtext = "Rules, guidance and support"
 
-  show_transition_link = true
+  show_transition_link = false
```

Comment:
should the COVID banner be hidden as well?

Reply:
Good question, we checked this with SMT and only the brexit banner should be removed.